driver/controller.go

```diff
@@ -191,6 +194,15 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	if snapshot != nil && vol != nil && volumeReq.SnapshotID != "" && int(volumeReq.SizeGigaBytes) > int(snapshot.SizeGigaBytes) && int(volumeReq.SizeGigaBytes) > 1 {
```

Comment:
Why do we check if the requested size is larger than one GB?

I think we don't need to check for any minimum values because the call in line 96 already looks for boundaries.

Reply:
IIRC it was because I was getting errors when trying to resize to a 1GB size since the minimal volume is 1GB. If you want to resize , you need to specify a size that is at least greater than 1GB or you will be doing a api/operation for nothing.